code-build.js

```diff
@@ -152,6 +152,8 @@ async function waitForBuildEndTime(
 
 function githubInputs() {
   const projectName = core.getInput("project-name", { required: true });
+  const disableSourceOverride =
+    core.getInput("disable-source-override") === "true";
```

Comment:
For consistency, could you either update this new line to use `core.getInput("disable-source-override", {required: false}) === "true"`, or remove all other `{ required: false }`s from githubInputs()?